src/Client.java

```diff
@@ -0,0 +1,54 @@
+import java.io.*;
+import java.net.Socket;
+
+public class Client
+{
+    private Socket socket;
+    private DataInputStream input;
+    private DataOutputStream output;
+
+    public Client(String ipAddress, int port)
+    {
+        try {
+            socket = new Socket(ipAddress, port);
+            input = new DataInputStream(new BufferedInputStream(socket.getInputStream()));
+            output = new DataOutputStream(socket.getOutputStream());
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private void send(String data)
+    {
+        try {
+            output.writeUTF(data);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private String receive()
+    {
+        String inputData = "";
```

Comment:
assignments should not interfere(cuddle) with loop
insert an empty line between them